ZCRMiOS/Classes/ZCRMModuleDelegate.swift

```diff
@@ -26,6 +26,11 @@ open class ZCRMModuleDelegate : ZCRMEntity
         return ZCRMRecordDelegate( recordId : id, moduleAPIName : apiName )
     }
     
+    public func getCustomView( cvName : String ) -> ZCRMCustomView
+    {
+        return ZCRMCustomView(cvName: cvName, moduleAPIName: self.apiName)
```

Comment:
This should hit request to return fully filled ZCRMCustomView object right?
If something will return an object without hitting request, it should return only delegate object.